src/engraving/libmscore/measure.cpp

```diff
@@ -3160,6 +3160,10 @@ void Measure::stretchMeasure(qreal targetWidth)
 
     std::multimap<qreal, Segment*> springs;
 
+    //Fraction minTicks = computeTicks();
+    Fraction minTicks = minSysTicks(); // The stretching must be done according to the shortest note *of the system*!
```

Comment:
This is overwrites the one a few lines up, causes a compiler warning. The commented out line above seems not needed